packages/combo-box/theme/lumo/vaadin-combo-box-item-styles.js

```diff
@@ -4,42 +4,18 @@ import '@vaadin/vaadin-lumo-styles/style.js';
 import { item } from '@vaadin/item/theme/lumo/vaadin-item-styles.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
-/* TODO partly duplicated from vaadin-list-box styles. Should find a way to make it DRY */
 const comboBoxItem = css`
   :host {
-    cursor: default;
-    -webkit-tap-highlight-color: var(--lumo-primary-color-10pct);
-    padding-left: calc(var(--lumo-border-radius-m) / 4);
-    padding-right: calc(var(--lumo-space-l) + var(--lumo-border-radius-m) / 4);
     transition: background-color 100ms;
-    border-radius: var(--lumo-border-radius-m);
     overflow: hidden;
     --_lumo-item-selected-icon-display: block;
   }
 
-  :host(:hover) {
-    background-color: var(--lumo-primary-color-10pct);
-  }
```

Comment:
After the change, the background-color rule will apply on hover only for not disabled items. Although it seems correct, I would ask for a written confirmation of that.

Here is the related rule from the `vaadin-item-styles` that will become effective after the change:
https://github.com/vaadin/web-components/blob/ab6f1cdef897340d16b2a9b76b8815345939ca2f/packages/item/theme/lumo/vaadin-item-styles.js#L67-L69

Reply:
This is actually intended. We do not have support for disabling combo-box items, but there is a workaround using `renderer` as I described here: https://github.com/vaadin/web-components/issues/1373#issuecomment-467372363, so it makes sense to take that into account.